actions/mmdebstrap_action.go

```diff
@@ -0,0 +1,177 @@
+/*
+mmdebstrap Action
+
+Construct the target rootfs with mmdebstrap tool.
+
+Please keep in mind -- file `/etc/resolv.conf` will be removed after execution.
+Most of the OS scripts used by `mmdebstrap` copy `resolv.conf` from the host,
+and this may lead to incorrect configuration when becoming part of the created rootfs.
+
+ # Yaml syntax:
+ - action: mmdebstrap
+   mirrors: <list of URLs>
+   suite: "name"
+   components: <list of components>
+   variant: "name"
+   keyring-packages:
+   keyring-files:
+
+Mandatory properties:
+
+- suite -- release code name or symbolic name (e.g. "stable")
+
+Optional properties:
+
+- mirrors -- list of URLs with Debian-compatible repository
+ If no mirror is specified debos will use http://deb.debian.org/debian as default.
+
+- variant -- name of the bootstrap script variant to use
+
+- components -- list of components to use for packages selection.
+ If no components are specified debos will use main as default.
+
+Example:
+ components: [ main, contrib ]
+
+- keyring-packages -- list of keyrings for package validation.
+
+- keyring-files -- list keyring files for repository validation.
+
+- merged-usr -- use merged '/usr' filesystem, true by default.
```

Comment:
As i understand it correctly; it's not true by default but rather follows the distribution default; so this seems incorrect

Reply:
I rephrased the help text in order to point out, that if the attribute is not specified it falls back to the distributions default.